privacyidea/lib/policy.py

```diff
@@ -971,6 +971,9 @@ def get_static_policy_definitions(scope=None):
             ACTION.AUDIT: {
                 'type': 'bool',
                 'desc': _('Allow the user to view his own token history.')},
+            ACTION.USERLIST: {'type': 'bool',
```

Comment:
Hi Cornelius,

I added this as the get_users() function in api/user.py has a prepolicy 
decorator that requires
the userlist action.

Maybe I have worked around a problem that should be fixed else where? 
This did occur to me when
I wrote the code..

Best Regards

Martin

On 2016-03-24 19:32, Cornelius Kölbel wrote:
> In privacyidea/lib/policy.py [1]:
> 
>> @@ -971,6 +971,9 @@ def get_static_policy_definitions(scope=None):
>>              ACTION.AUDIT: {
>>                  'type': 'bool',
>>                  'desc': _('Allow the user to view his own token
> history.')},
>> +            ACTION.USERLIST: {'type': 'bool',
> 
> In my opinion this makes no sense
> 
> --
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub [2]
> 
> 	*
> 
> Links:
> ------
> [1] 
> https://github.com/privacyidea/privacyidea/pull/345#discussion_r57374796
> [2]
> https://github.com/privacyidea/privacyidea/pull/345/files/f64bfe1a658f9e19818e9a58430fb91b1a3868f5#r57374796